ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@ const Yaml = (props: IProps) => {
                   />
                 </Space>
               </div>
```

Comment:
The ref attribute should be moved to the parent div if it's intended to reference the entire content area.